pinot-core/src/test/java/org/apache/pinot/queries/NullEnabledQueriesTest.java

```diff
@@ -317,16 +395,17 @@ public void testQueries(Number baseValue, ColumnDataType dataType, boolean nullV
           //(3 rows)
           assertEquals(row[3], 0L);
         } else if ((int) row[keyColumnIdx] == 1) {
-          assertTrue(Math.abs(((Double) row[0]) - 4 * _sumKey1) < 1e-1);
-          assertTrue(Math.abs(((Double) row[1]) - baseValue.doubleValue()) < 1e-1);
+          assertTrue(Math.abs(((Double) row[0]) - 4 * _sumKey1) < PRECISION);
```

Comment:
I saw the value of `PRECISION` constant is `1` in this PR, isn't it greater than `1e-1`?

Reply:
yeah this was for integer and long in the future. but yes i can make it 1e-1 for now until we add other types